libpod/logs/log.go

```diff
@@ -162,6 +168,24 @@ func getTailLog(path string, tail int) ([]*LogLine, error) {
 	return tailLog, nil
 }
 
+//getColor returns a ANSI escape code for color based on the ColorID
+func getColor(ColorID int64) string {
+	colors := map[int64]string{
+		0: "\033[37m", // Light Gray
+		1: "\033[31m", // Red
+		2: "\033[33m", // Yellow
+		3: "\033[34m", // Blue
+		4: "\033[35m", // Magenta
+		5: "\033[36m", // Cyan
+		6: "\033[32m", // Green
+	}
+	return colors[ColorID%int64(len(colors))]
+}
+
+func (l *LogLine) colorize(prefix string) string {
+	return getColor(int64(l.ColorID)) + prefix + l.Msg + ANSIEscapeResetCode
```

Comment:
why cast to int64 here? `l.ColorID` is already int64, no?

Reply:
Indeed, this does not need casting.